src/gradient.class.js

```diff
@@ -307,7 +344,7 @@
      * @see http://www.w3.org/TR/SVG/pservers.html#LinearGradientElement
      * @see http://www.w3.org/TR/SVG/pservers.html#RadialGradientElement
      */
-    fromElement: function(el, instance, opacityAttr) {
+    fromElement: function(el, instance, opacityAttr, svgOptions) {
```

Comment:
You might as well add a javadoc line for svgOptions.

Reply:
sure, i m not done yet, svg export is now broken and needs to be adequate to the new logic.